etc/kayobe/ansible/filter_plugins/filters.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+from ansible.errors import AnsibleFilterError
+
+class FilterModule(object):
+    def filters(self):
+        return {
+            'group_hostvars_by_var':
+            self.group_hostvars_by_var,
+            'group_hostvars_by_host':
+            self.group_hostvars_by_host
+        }
+
+    def group_hostvars_by_var(self, hostvars, var, stdout=None):
+        """
+        Returns a dictionary where the keys are values for the
+        specified var in hostvars, and the values are the hosts
+        that match that value.
+        """
+        result = {}
+
+        for host in hostvars.keys():
+            try:
+                indiv_host_all = hostvars[host]
+                indiv_host_var = indiv_host_all[var]
```

Comment:
Can be done in one jump
```suggestion
                indiv_host_var = hostvars[host][var]
```